doc/install.md

```diff
@@ -20,8 +21,29 @@ If you do not have a cluster yet, create one by using one of the following provi
 * [kubeadm](https://kubernetes.io/docs/admin/kubeadm/)
 * [minikube](https://minikube.sigs.k8s.io/)
 
-## Install release v0.19.8 and use Coscheduling
-
+## Install release v0.19.8 as a second scheduler and use Coscheduling
```

Comment:
Some questions:
Do we really need to guide the user to deploy the scheduler-plugins as a secondary scheduler？
1. If we deploy two scheduler in the same cluster, there must be resource conflicts although we can isolate resources by label. 
2. Users will change `.spec.schedulerName` to match the secondary scheduler. This will be more complicated if the user has a lot of yaml.
3. If we stop the default scheduler after we deploy the secondary scheduler. Other discussion here: https://github.com/kubernetes-sigs/scheduler-plugins/pull/142#discussion_r565806750
4. Compared with other schedulers, I think the advantage of scheduler plugins is that it can completely replace the default scheduler.

So I think we need to add helm chart to replacing the default-scheduler more conveniently other than deploy it as a secondary scheduler.

@FingerLiu @Huang-Wei

Reply:
@denkensk using one scheduler is our ultimate goal. But for now, that needs access to the control plane, which can be a high blocker for users who want to try out scheduler framework. So running the scheduler plugins as a secondary scheduler is a good start to get them started, but as you mentioned, not the most optimal solution.

So generally we'd like to provide instructions for both:

- running scheduler-plugins as a single scheduler (for now, it's not fully automated by Helm chart)
- running scheduler-plugins as a secondary scheduler - can be automated by Helm chart

Re 2: an option is to keep workloads yamls as is, and set up an admission webhook to mutate `.spec.schedulerName` on demand.

Reply:
> So running the scheduler plugins as a secondary scheduler is a good start to get them started, but as you mentioned, not the most optimal solution.

Thanks. Easy to start.  I agree with this. 
Can we add some additional notes to tell users: Try out scheduler framework by deploying secondary scheduler. But if in the production environment, it is recommended to replacereplacing the default-scheduler manually.
@Huang-Wei @FingerLiu

Reply:
Sure, we should add some notes there.

Reply:
OK, I will add these notes.

Reply:
added this note ".**But if in the production environment, it is recommended to replace the default-scheduler manually(as described in next chapter).**"